### PR TITLE
Add blurb to invite modal to explain that invite links expire

### PR DESCRIPTION
**Pull request by rowanseymour**
Captura de pantalla 2024-04-24 a la(s) 09 53 24

### CODE REVIEW

**Comment by rowanseymour** [comment]
@norkans7 I don't necessarily disagree with putting an expiry date on this page but not obvious where...

Captura de pantalla 2024-04-24 a la(s) 09 52 25

**Comment by norkans7** [comment]
we can try `PENDING INVITATIONS (expire in 30 days)` in that label/title?

**Comment by rowanseymour** [comment]
I think that's confusing because it's a list of things which will expire at different times. 

I don't think this is a big problem.. nobody should be expecting invitations to hang around for ever.

File: temba/settings_common.py [hunk 1]
```diff
@@ -698,6 +698,8 @@
 
 ANONYMOUS_USER_NAME = "AnonymousUser"
 
+INVITATION_VALIDITY = timedelta(days=30)
```
[the remark this answers is not in the document]
Comment (norkans7) [reply temba/settings_common.py:701]: Okay, I was thinking that but I do not see why we keep the invite around

**Review by norkans7 — APPROVED** [review]

File: temba/settings_common.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rowanseymour) [reply temba/settings_common.py:701]: Same reason we keep anything around instead of hard deleting it - historical record

**Pull request merged**